packages/config/src/networks.ts

```diff
@@ -62,4 +65,8 @@ export const genesisData: Record<NetworkName, GenesisData> = {
     genesisTime: 1665396300,
     genesisValidatorsRoot: "0x9d642dac73058fbf39c0ae41ab1e34e4d889043cb199851ded7095bc99eb4c1e",
   },
+  ephemery: {
+    genesisTime: ephemeryChainConfig.MIN_GENESIS_TIME + ephemeryChainConfig.GENESIS_DELAY,
+    genesisValidatorsRoot: "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
From the spec
> It's important to note that genesis_validators_root is normally predefined in the client but in this case it's not known in advance which can potentially break certain architectures. For example light clients which are relying on hardcoded genesis_validators_root won't work.

Not sure how to resolve this issue, @g11tech maybe you have some thoughts on this.